src/components/TextField/TextField.tsx

```diff
@@ -33,6 +33,7 @@ interface TextFieldProps extends VibeComponentProps {
   onBlur?: (event: React.FocusEvent) => void;
   onFocus?: (event: React.FocusEvent) => void;
   onKeyDown?: (event: React.KeyboardEvent) => void;
+  onWheel?: (event: React.MouseEvent) => void;
```

Comment:
@or-nuri-monday `React.WheelEvent` would be more accurate

Reply:
Changing now